### ISSUE TITLE
[BUG] call post build plugin <class 'pyarmor.cli.plugin.DarwinUniversalPlugin'>

### ISSUE THREAD

**Issue by knro** [dashingsoft/pyarmor#1430]
8.3.3 (Pro)
Platform: linux.x86_64

When I run this command:

```
pyarmor gen -i foo
```

All steps goes OK until it fails here:

```
INFO     call post build plugin <class 'pyarmor.cli.plugin.DarwinUniversalPlugin'>
ERROR    [Errno 2] No such file or directory: 'dist/pyarmor_runtime_xxxxxxx'
```

Inside dist, I can see foo generated fine and inside of it I see the correct pyarmor_runtime_xxxxxxx

The `DarwinUniversalPlugin` is kind of suspicious, does it think I'm on MacOS where I am on Linux?

**Comment by jondy** [comment]
This is a bug, it will be fixed in next release.

`DarwinUniversalPlugin` should do nothing in this case

As a quick workaround, disable this plugin by this command
```
pyarmor cfg plugins = ""
```

**Comment by jondy** [comment]
Fixed in 8.3.4
